packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -610,60 +681,18 @@ export const useTransactionHistory = (
     typeof txPages !== 'undefined' &&
     data.length === txPages.flat().length
 
-  // transfers initiated by the user during the current session
-  // we store it separately as there are a lot of side effects when mutating SWRInfinite
-  const { data: newTransactionsData, mutate: mutateNewTransactionsData } =
```

Comment:
`newTransactionData` (overall renamed to "current session transactions" which is easier to understand what it's about imo) is removed from here, same as relevant logic below.

New hook, `useCurrentSessionTransactions`, handles these transactions separately because it's independent of all the partial transactions we fetch.